locales/en.json

```diff
@@ -71,28 +71,28 @@
 	"Header": "Header",
 	"Unordered List": "Unordered List",
 	"Ordered List": "Ordered List",
-	"Todo List": "Todo List",
+	"Todo List": "Checklist",
 	"Blockquote": "Blockquote",
 	"Bold font": "Bold font",
 	"Italics font": "Italics font",
 	"Strikethrough": "Strikethrough",
-	"Inserted text": "Inserted text",
-	"Marked text": "Marked text",
+	"Inserted text": "Underlined text",
+	"Marked text": "Highlighted text",
 	"Link": "Link",
 	"Image": "Image",
 	"Code": "Code",
 	"Externals": "Externals",
-	"This is a alert area.": "This is a alert area.",
+	"This is a alert area.": "This is an alert area.",
 	"Revert": "Revert",
 	"Import from clipboard": "Import from clipboard",
 	"Paste your markdown or webpage here...": "Paste your markdown or webpage here...",
 	"Clear": "Clear",
 	"This note is locked": "This note is locked",
-	"Sorry, only owner can edit this note.": "Sorry, only owner can edit this note.",
+	"Sorry, only owner can edit this note.": "Sorry, only the owner can edit this note.",
 	"OK": "OK",
 	"Reach the limit": "Reach the limit",
-	"Sorry, you've reached the max length this note can be.": "Sorry, you've reached the max length this note can be.",
-	"Please reduce the content or divide it to more notes, thank you!": "Please reduce the content or divide it to more notes, thank you!",
+	"Sorry, you've reached the max length this note can be.": "Sorry, you've reached the maximmum length this note can be.",
+	"Please reduce the content or divide it to more notes, thank you!": "Please shorten the file.",
```

Comment:
As we use "note" everywhere else, I agree that "note" is maybe the best word here. As an alternative to content and file, I would suggest "document".

I would like to avoid the word "file" because in the world of computers it's associated with the elements in your "explorer" and describes pictures as well as whole databases (for example in the case of SQLite).

The word "document" is used by Microsoft as well as Google and LibreOffice in their writing apps to describe the file as well as the content of it. Which makes it a good candidate, because the users already know it.

But as already mentioned to not confuse the users I would stay with "note" as this is the word everywhere else is also used.